.github/workflows/release-snapshot.yml

```
@@ -0,0 +1,82 @@
+name: Release Snapshot
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release-snapshot:
+    name: Release Snapshot
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)